Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM lancachenet/ubuntu-nginx:latest
-MAINTAINER LanCache.Net Team <team@lancache.net>
+FROM lancachenet/monolithic:switcheroo
```

Comment:
One option to make this more flexible:
```dockerfile
ARG MONOLITHIC_TAG=master
FROM lancachenet/monolithic:${MONOLITHIC_TAG}
```

We could potentially leverage the current branch name with this (e.g. attempt to pull `lancachenet/monolithic:$GITHUB_HEAD_REF`, if that works then pass `--build-arg MONOLITHIC_TAG="${GITHUB_HEAD_REF}"` when building the image; this would allow automatically leveraging matched-name branches on `monolithic` for pull requests on `generic`).

Reply:
I like this, but don't want to make the change now